server/controllers/finance/fiscal.js

```diff
@@ -98,13 +107,21 @@ function list(req, res, next) {
   }
 
   filters.setOrder(ordering);
-
   const query = filters.applyQuery(sql);
   const parameters = filters.parameters();
 
+  let fiscals = [];
   db.exec(query, parameters)
     .then((rows) => {
-      res.status(200).json(rows);
+      fiscals = rows;
+      return q.all(fiscals.map(fiscal => {
```

Comment:
This has the potential to send back a lot of data!  If we do this, it would be better to put it behind a query string flag - like `?includePeriods=1`.

That being said, I much prefer the solution using `FiscalService.getPeriods()` as a second HTTP call.

Reply:
Okey! adding a parameter is much better